public/views/FileTransfer.svelte

```diff
@@ -368,6 +367,29 @@
 
   $: watchUsersCount(usersCount, percentage);
 
+  function dropHandler(ev) 
+  {
+    var files = ev.dataTransfer.files;
+    if (files != null)
+    {
+      document.getElementById('drop_zone').style.border = 'dotted';
+      selectFiles(files);
+    }
+  }
+
+  function dragOverHandler(ev) 
+  {
+    document.getElementById('drop_zone').style.border = 'solid';
+  }
+  function dragLeaveHandler(ev) 
+  {
+    document.getElementById('drop_zone').style.border = 'dotted';
+  }
+  function changeHandler(ev)
+  {
+    selectFiles(ev.target.files);
+  }
```

Comment:
These functions are redundant and can be made inline in the `drop_zone` element itself.

Reply:
Hi, how could i make a "border style change" inline in the drop_zone ? I'am new in Svelte and i cannot find a good solution. I have already redesigned the third function, but i have a problem with the first two. Could you please help me with this problem ?

Reply:
Sure @adam0k. 
This is more of a JavaScript concept than Svelte itself, and I think you kind of figured it out in the latest commit.

Right now, this is the code
```
<div
    id="drop_zone"
    ...
    on:dragover|preventDefault={ev => {document.getElementById('drop_zone').style.border = "solid";} }
/>
```

But we can refactor it and write it like this using Svelte Modifiers and `ev.target` property

```
<div
    id="drop_zone"
    ...
    on:dragover|preventDefault|self={e => {e.target.style.border = 'solid';}}
/>
```

Similarly for other properties, you can refactor to match the above syntax. This way we don't have to keep the old functions since they won't be used anywhere apart from this place.
Another better way to solve this problem would be to make use of **State variables** in Svelte. They will make the code shorter and more logical, but I think your method is also fine :smile:

Reply:
thx for explanation, I add new commit with with adequate changes ;)